File: Solutions/JavaScript/pattern012.js
// Pattern Programs - 012 [JavaScript]
// Github Repository: https://github.com/mxsena/pattern-programs
let pattern = ""
let n = 5
let x

for(let i=1; i<=n; i++){
    x = i
    for(let j=1; j<=n; j++){
        pattern += x.toString().padStart(3," ")
        x += n
    }
    pattern += "\n"
}
console.log(pattern)

File: Solutions/C++/pattern012.cpp
// Pattern Programs - 012 [C++]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <iostream>
using namespace std;

int main(void)
{
    int n = 5;
    int x;
    
    for (int i = 1; i <= n; i++)
    {
        x = i;
        for (int j = 1; j <= n; j++)
        {
            cout << x << " ";
            x += n;
        }
        cout << endl;
    }
    return 0;
}